dev/tasks/macros.jinja

```diff
@@ -182,15 +182,19 @@ env:
 {% endmacro %}
 
 {%- macro github_upload_wheel_scientific_python(pattern) -%}
-  {%- if arrow.is_default_branch() -%}
   - name: Upload wheel to Anaconda scientific-python
     shell: bash
     run: |
+      # check if completion actually expands to wheel files
+      # to prevent empty dirs from failing silently
+      if ! test -n "$(find . -wholename '{{ pattern }}' -print -quit)"; then
```

Comment:
We may be able to simplify this:

```suggestion
      if [ "$(echo {{ pattern }})" = "{{ pattern }}" ]; then
```

Reply:
No I don't think that works (it doesn't locally for me) and would also not work if nullglob is set etc. find is less brittle.